### ISSUE TITLE
BUG: Fatal error when using withCount()

### ISSUE THREAD

**Issue by weidmaster** [itsgoingd/clockwork#425]
I tried to used ```withCount()``` Eloquent method and I got a fatal error from clockwork. Here is the error screenshot. I also get the same error from ```artisan tinker```

The code works as expected, so the issue is with clockwork.

Eloquent call:
```php
$brands = Brand::where('status', 1)->withCount('products')->orderBy('name')->get();
```
Correct result with debug false (no clockwork):
![Captura de Tela 2020-09-21 às 17 21 02](https://user-images.githubusercontent.com/9614642/93817190-f528ac00-fc2e-11ea-9819-82f2f475998b.png)

I am using Laravel 5.6.
<img width="1440" alt="Captura de Tela 2020-09-21 às 17 14 04" src="https://user-images.githubusercontent.com/9614642/93817273-15f10180-fc2f-11ea-9a3d-af03dcd606e0.png">

![Captura de Tela 2020-09-21 às 17 20 44](https://user-images.githubusercontent.com/9614642/93817289-1d180f80-fc2f-11ea-9b53-c4b6b42659a7.png)

Now for my application to work I have to disable the debug in my .env file, but then I don't get the clockwork experience.

Please take a look and if there is a workaround, let me know.

Thanks in advance.

**Comment by itsgoingd** [comment]
Hey, this should be fixed in Clockwork 4.1.7, if you are not running the latest version please try updating.

**Comment by weidmaster** [comment]
@itsgoingd Oh that fixed the issue. I didn't know I could update just this package with composer. It was in version 4.1.6 and now it is in 4.1.8 and it works! Thank you for fixing. I love this package.
